src/api/utils/txs/evaluate/model.rs

```diff
@@ -0,0 +1,359 @@
+            "additionalUtxoSet": [
+                [
+                  {
+                    "txId": "stringstringstringstringstringstringstringstringstringstringstri",
+                    "index": 4294967295u64
+                  },
+                  {
+                    "address": "addr_test1qz66ue36465w2qq40005h2hadad6pnjht8mu6sgplsfj74qdjnshguewlx4ww0eet26y2pal4xpav5prcydf28cvxtjqx46x7f",
+                    "value": {
+                      "coins": 2,
+                      "assets": {
+                        "3542acb3a64d80c29302260d62c3b87a742ad14abf855ebc6733081e": 42,
+                        "b5ae663aaea8e500157bdf4baafd6f5ba0ce5759f7cd4101fc132f54.706174617465": 1337
+                      }
+                    },
+                    "datumHash": "iamadatumhash",
+                    "datum": "base16datum",
+                    "script": {"native": {"all":[{"startsAt": 1234567890u64},"ec09e5293d384637cd2f004356ef320f3fe3c07030e36bfffe67e2e2","3c07030e36bfff7cd2f004356ef320f3fe3c07030e7cd2f004356437"]}}
+                  }
+                ]
+                ]
+        });
+
+        let req: TxEvaluationRequest = serde_json::from_value(value).unwrap();
+        assert_eq!(req.cbor, "enocodedcbor");
+        let utxo_set = req.additional_utxo_set.unwrap();
+        assert_eq!(utxo_set.len(), 1);
+        assert_eq!(
+            utxo_set[0].0.tx_id,
+            "stringstringstringstringstringstringstringstringstringstringstri"
+        );
+        assert_eq!(utxo_set[0].0.index, 4294967295u64);
+        assert_eq!(
+            utxo_set[0].1.address,
+            "addr_test1qz66ue36465w2qq40005h2hadad6pnjht8mu6sgplsfj74qdjnshguewlx4ww0eet26y2pal4xpav5prcydf28cvxtjqx46x7f"
+        );
+        assert_eq!(utxo_set[0].1.value.coins, 2);
+    }
+
+    #[test]
+    fn test_json_request_plutusv2() {
+        let value = json!({
+            "cbor": "enocodedcbor",
+            "additionalUtxoSet": [
+                [
+                  {
+                    "txId": "stringstringstringstringstringstringstringstringstringstringstri",
+                    "index": 4294967295u64
+                  },
+                  {
+                    "address": "addr_test1qz66ue36465w2qq40005h2hadad6pnjht8mu6sgplsfj74qdjnshguewlx4ww0eet26y2pal4xpav5prcydf28cvxtjqx46x7f",
+                    "value": {
+                      "coins": 2,
+                      "assets": {
+                        "3542acb3a64d80c29302260d62c3b87a742ad14abf855ebc6733081e": 42,
+                        "b5ae663aaea8e500157bdf4baafd6f5ba0ce5759f7cd4101fc132f54.706174617465": 1337
+                      }
+                    },
+                    "datumHash": "iamadatumhash",
+                    "datum": "base16datum",
+                    "script": {"plutus:v2": "base16script"}
+                  }
+                ]
+                ]
+        });
+
+        let _req: TxEvaluationRequest = serde_json::from_value(value).unwrap();
+    }
+}
+*/
```

Comment:
Should this file be commented out and committed?

It also looks like we don't have `treefmt` run on this, since it's missing a trailing newline.

Reply:
Fixed in b4b535e552d52a2541d81728db47811dc9b1ace8